### ISSUE TITLE
Bplan not shown on mB, statement form broken

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4824]
**URL:** https://www.berlin.de/ba-spandau/politik-und-verwaltung/aemter/stadtentwicklungsamt/stadtplanung/bebauungsplaene/bebauungsplan.1280937.php
**user:** user of both meinBerlin project overview and berlin.de statement form
**expected behaviour:** bplan is correctly shown in project overview, on imperia the statement form works
**behaviour:** bplan is not shown (draft without tile picture), client cannot use statement form (on chrome for mac it works, see screenshot)
**important screensize:**
**device & browser:** more information on client browser has been asked for
**Comment/Question:** Bezeichner ist correct and shown on Bplan-map.

Screenshot?
<img width="1027" alt="Bildschirmfoto 2023-01-09 um 10 17 24" src="https://user-images.githubusercontent.com/35491681/211274922-aa78cc05-701e-4bc1-b6e4-7d7918ea03a0.png">

**Comment by CarolingerSeilchenspringer** [comment]
initiator manually resolved the problem but it happens regularly so should find a solution at some point. Will leave issue open but changed the prio to low.

**Comment by fuzzylogic2000** [comment]
@CarolingerSeilchenspringer But isn't the issue that the bplan project is not published (= still draft) after adding it via imperia? It doesn't have anything to do with the statetment form? (If anyting, the statement form shouldn't work at all when the bplan is not published, but that's not what we are meant to fix here, right?)
@goapunk had the idea of adding more info to the post. I think that would be good, to have a better idea what happened next time. Can we either add a story for that or add a task to the release story? Or maybe rename this issue and make it say what we actually want here.

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 I still cannot say what the problem is as long as it is always fixed manually before we get a chance to look at it so in this issue I described the bug as it has been described to me. We shouldn't investigate without an actual case. I set the tag from Prio:low to Status:Blocked so we can get back to it when it happens again.

**Comment by fuzzylogic2000** [comment]
Yes, but if we wait until it accurs again, we will again not have enough info to act on it. The idea was to add more info now, so that we have all that info when it does happen again. Does that make sense?
Also, it is most likely not on our end, because the API behaves like it should and published the bplans when it should.

**Comment by goapunk** [comment]
Adding some print statements which log the request they make should be very easy and help us identify the issue, so maybe we should do it as part of the release?

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 @goapunk if it is easy and helps then lets do it. But prio stays low for now. I will move to prod release milestone as that gives us a little more time after the demo release and we can focus on the stories first.
